index.html

```diff
@@ -201,7 +203,7 @@ <h1 class="gameCardTitle" id="games_genre_rank_3_title">Title #3</h1>
                                             </div>
                                         </div>
     
-                                        <div style="height: 55px; background-color: #CD7F32;" class="podiumStep">3</div>
+                                        <div style="background-color: #CD7F32;" class="podiumStep">3</div>

```

Comment:
The problem that there is no difference between the 1st, 2nd and third place, You have to let the size in the style here.